snapshot/overlay/overlayfs.go

```diff
@@ -1,4 +1,4 @@
-package snapshot
+package overlay
 
```

Comment:
nit: I wonder the file should be renamed to overlay.go, and the structure Overlayfs should be renamed to Overlay.